### PR TITLE
`<IconButton/>`- adding a "transparent" & "premium" skins

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Adding an example of a `transparent` and `premium` skins for both `primary` and `secondary` priorities. 

This PR includes: 
1. Updating documentation
2. Updating `IconButton` API
3. Adding `transparent`& `premium` skins to e2e and story
4. Removing e2e tests which already exist in visual tests. 

Related to this issue: #3556

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
Hey @LittleTal ,
I've added the `transparent` examples for the `<IconButton/>`:
https://wix-wix-style-react-pr-3624.surge.sh/?selectedKind=5.%20Buttons&selectedStory=5.2%20IconButton&full=0&addons=0&stories=1&panelRight=0

Can you please have a look? :)

File: src/IconButton/docs/index.story.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export default {
               {
                 title: 'Skin',
                 text:
-                  'Icon Button supports 3 skin styles. `standard` works well on white, `inverted` is great on grey and `light` is perfect on dark backgrounds.',
+                  'Icon Button supports 4 skin styles. `standard` works well on white, `inverted` is great on grey, `light` is perfect on dark backgrounds and `transparent`.',
```
Comment (NitayRabi) [inline src/IconButton/docs/index.story.js]: What `transparent` is good for?

File: src/IconButton/test/IconButton.e2e.js [hunk 2]
```diff
@@ -18,10 +18,6 @@ describe('IconButton', () => {
       await checkTestStory(testStories.ICONBUTTON_SKINS);
     });
 
-    eyes.it('check iconbutton affixes', async () => {
-      await checkTestStory(testStories.ICONBUTTON_SIZES);
-    });
```
Comment (NitayRabi) [inline src/IconButton/test/IconButton.e2e.js]: Why is this removed?

File: src/IconButton/test/testStory.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ const TestContainer = ({ children }) => (
   </div>
 );
 
-const skins = ['standard', 'inverted', 'light'];
+const skins = ['standard', 'inverted', 'light', 'transparent'];
```
Comment (NitayRabi) [inline src/IconButton/test/testStory.js]: Create `Skins` constant for this strings

File: src/IconButton/constants.js [hunk 4]
```diff
@@ -1,6 +1,25 @@
-export const iconButtonSizes = {
+export const iconButtonChildSize = {
   tiny: { size: 'tiny', px: '18px' },
   small: { size: 'small', px: '18px' },
   medium: { size: 'medium', px: '24px' },
   large: { size: 'large', px: '24px' },
 };
+
```
Comment (NitayRabi) [inline src/IconButton/constants.js]: Better usage IMO - 
```javascript
export const iconButtonChildSize = {
	[Sizes.tiny]: '18px',
	...
	...
}
```
